Sources/SwiftParser/TokenSpecSet.swift

```diff
@@ -414,6 +415,7 @@ enum DeclarationModifier: TokenSpecSet {
     case TokenSpec(.static): self = .static
     case TokenSpec(.unowned): self = .unowned
     case TokenSpec(.weak): self = .weak
+    case TokenSpec(.resultDependsOnSelf): self = .resultDependsOnSelf
```

Comment:
You should check if the experimental feature is enabled, similar to here

https://github.com/apple/swift-syntax/blob/1fa18e5b5ee89eb1702f55ca3059e0a405e9cc90/Sources/SwiftParser/generated/Parser%2BTokenSpecSet.swift#L2323